src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -152,6 +177,10 @@ public double getPosition() {
     );
   }
 
+  /**
+   * set the desired power to the m_desiredPower variable
+   * @param power
+   */
```

Comment:
maybe just say it sets the motor power for the manual control mode. mostly in javadocs, you aren't explaining how something happens, but how to use it/what the effect is